build_tools/ci/cpu_comparison/run.py

```diff
@@ -1123,9 +1123,34 @@ def all_tests(
     parser.add_argument("peano_install_dir", type=abs_path)
     parser.add_argument("--xrt-dir", type=abs_path)
     parser.add_argument("--vitis-dir", type=abs_path)
-    parser.add_argument("--xrt_lite_n_core_rows", type=int)
-    parser.add_argument("--xrt_lite_n_core_cols", type=int)
-    parser.add_argument("--target_device", type=str, required=True)
+    parser.add_argument(
+        "--xrt_lite_n_core_rows",
+        type=int,
+        help="Number of AIE core rows of the xrt-lite device to use",
+    )
+    parser.add_argument(
+        "--xrt_lite_n_core_cols",
+        type=int,
+        help="Number of AIE core columns of the xrt-lite device to use",
+    )
+
+    # Taken from AMDAIEEnums.td
+    current_devices = [
+        "xcvc1902",
+        "xcve2302",
+        "xcve2802",
+        "npu1",
+        "npu1_1col",
+        "npu1_2col",
+        "npu1_3col",
+        "npu1_4col",
+        "npu4",
+    ]
```

Comment:
I'm not sure if we've tested for all these devices.

Would it be okay to just go with `npu1_4col` and `npu4` as the two accepted devices ?

For instance, `run_on_target` has been defaulted to be `npu1_4col` for all tests and just one test adds `npu4` to the same list.

So, there should just be a correspondence between the two, I believe.

Reply:
Okay okay, that makes sense.